README.md

```diff
@@ -74,3 +74,13 @@ with enough entropy, that library uses the [randombytes library](https://www.npm
 cryptographically strong random values. The use of the [bip39 library](https://www.npmjs.com/package/bip39) is limited to
 the creation of mnemonic phrases, the code that creates the binary seeds is in
 [skycoin-lite](https://github.com/skycoin/skycoin-lite).
+
+## Pre-release testing
+
+Performs these actions before releasing:
+
+- Test if all options are working correctly when opening the wallet on the test server (using `npm start`). Load a wallet with balance and try to send coins, to verify if the functions for loading wallets and creating of transactions work correctly.
+
+- Test if all options are working correctly when opening the wallet from the local file system (the compilation is done by executing `make build-for-local-fs`). In the local version it is important to check all the pages and make sure that all the images load correctly. For this it is useful to go through all pages while the development console is open.
+
+- Create/load a wallet in a production build (it can be the one created by `make build-for-local-fs`) and then refresh the browser. The wallet should return to the wizard, which means that the data is not kept permanently.
```

Comment:
This step has an e2e test so it doesn't have to be here, what do you think?

Reply:
You are right. When I wrote the notes I had not yet uploaded the commit with the e2e test and forgot to update the text.